helm/g2p-sandbox/values.yaml

```diff
@@ -146,7 +155,20 @@ ph-ee-engine:
     ams_local_customer_host: "https://fynams.sandbox.mifos.io/"
     ams_local_auth_host: "https://fynams.sandbox.mifos.io/"
     ingress:
-      enabled: false    
+      enabled: true
+      annotations: 
+        kubernetes.io/ingress.class: nginx
+      tls:
+        - secretName: sandbox-secret
+      hosts:
+        - host: ams-mifos.sandbox.fynarfin.io          
```

Comment:
keep this as mifos and change from cookbooks

Reply:
Changed